collaborative-groth16/src/groth16.rs

```diff
@@ -85,115 +57,14 @@ use g=q^t (this is a 2^s-th root of unity) as (some kind of) generator and compu
 then if log2(domain_size) equals s we take as root of unity q^2, and else we take the log2(domain_size) + 1-th element of the domain created above
 */
 fn root_of_unity_for_groth16<F: PrimeField + FftField>(domain: &GeneralEvaluationDomain<F>) -> F {
-    let (q, roots) = roots_of_unity::<F>();
+    let (q, roots) = circom_snarks::roots_of_unity::<F>();
```

Comment:
this should probably live in a utils module in `circom_snarks`

Reply:
now lives under co_circom_snarks::utils